wpilibcExamples/src/test/cpp/examples/ArmSimulation/cpp/ArmSimulationTest.cpp

```diff
@@ -50,11 +50,11 @@ class ArmSimulationTest : public testing::TestWithParam<units::degree_t> {
 TEST_P(ArmSimulationTest, Teleop) {
   EXPECT_TRUE(frc::Preferences::ContainsKey(kArmPositionKey));
   EXPECT_TRUE(frc::Preferences::ContainsKey(kArmPKey));
-  EXPECT_DOUBLE_EQ(kDefaultArmSetpoint.value(),
-                   frc::Preferences::GetDouble(kArmPositionKey, NAN));
-
   frc::Preferences::SetDouble(kArmPositionKey, GetParam().value());
   units::degree_t setpoint = GetParam();
+  EXPECT_DOUBLE_EQ(GetParam().value(),
```

Comment:
```suggestion
  EXPECT_DOUBLE_EQ(setpoint.value(),
```
The variable is declared in the previous line, and should be used